Program.cs

```diff
@@ -25,6 +25,8 @@ public static async Task<int> Main(string[] args)
 
       return await Bootstrapper.Factory
         .CreateWeb(args)
+        .AddSetting(Keys.Host, "dotnetfoundation.org")
+        .AddSetting(Keys.LinksUseHttps, true)
```

Comment:
silly question, but perhaps https ought to be the default? :)

Reply:
These days it probably should be, and maybe Statiq is the right time to break the old behavior (this setting is a carryover from Wyam - changing it there would have quietly broken everyone who wasn’t HTTPS)